Assets/Scripts/Nav/Destination/NavTerrainDestinationSystem.cs

```diff
@@ -20,17 +25,21 @@ protected override void OnCreate()
 
         protected override void OnUpdate()
         {
+            var collisionWorld = buildPhysicsWorld.PhysicsWorld.CollisionWorld;
```

Comment:
Not sure when to us e CollisionWorld vs PhysicsWorld, but the [docs examples](https://docs.unity3d.com/Packages/com.unity.physics@0.4/manual/collision_queries.html) are in CollisionWorld so thats what I've used here, but noticed the rest of your project was using PhysicsWorld, so I can change this to be consistent if you would prefer.

Reply:
Functionally I'm not sure what the difference is between the two, and have been wondering about that as well. I may have to dig into Unity's code to find out. One may be more performant than the other, but I'm not sure which. For consistency with this project, you can just use the physics world, but if it turns out using the collision world is objectively better somehow, then we can always refactor everything in one fell swoop later.

Reply:
Agreed - updated.